quinn-proto/src/connection/mod.rs

```diff
@@ -1720,7 +1720,7 @@ where
 
         space
             .crypto_stream
-            .insert(crypto.offset, crypto.data.clone());
+            .insert(crypto.offset, crypto.data.clone(), crypto.data.len());
```

Comment:
I think we should instead pass down the `payload.len()` from `process_payload()` into `read_crypto()` and use that here, does that make sense to you?

Reply:
Yeah, makes sense. Fixed it.